06_puppy/brvishnu92/main_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"bytes"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type StorerTestSuite struct {
+	suite.Suite
+	storer Storer
+}
+
+func (suite *StorerTestSuite) SetupTest() {
+	tempPuppy := Puppy{id: 1, breed: "cat", color: "Black", value: "val1"}
+	suite.storer.CreatePuppy(tempPuppy)
+}
+
+func (suite *StorerTestSuite) TestCreate() {
+	crePup := Puppy{id: 1, breed: "cat", color: "Black", value: "val1"}
+	suite.storer.CreatePuppy(crePup)
+	returnPup := suite.storer.ReadPuppy(crePup.id)
+	suite.Equal(returnPup.id, crePup.id)
+	suite.Equal(returnPup.breed, crePup.breed)
+}
+
+func (suite *StorerTestSuite) TestRead() {
+	ExistingPup := *suite.storer.ReadPuppy(1)
```

Comment:
is this supposed to be a global variable?

Reply:
Done. Thanks 👍